core/space/domain/domain.go

```diff
@@ -26,6 +26,7 @@ type ThreadInfo struct {
 type FileInfo struct {
 	DirEntry
 	IpfsHash string
+	BackedUp bool
```

Comment:
@jsonsivar 

What do you think about renaming this to `Backup` so we have the same convention as in the `Bucket` `struct`?